packages/now-next/src/index.ts

```diff
@@ -378,18 +379,12 @@ export const build = async ({
     }
 
     const resultingExport = await getExportStatus(entryPath);
-    if (!resultingExport) {
+    if (!resultingExport || resultingExport.success !== true) {
```

Comment:
These errors were intentionally separate.

If the file is missing, there's a Next.js bug (truthy check) -- this should be considered an invariant.

If the file exisits and success is false, their pages have a bug.

Reply:
Broke them back a part